### PR TITLE
Fixes checkpointing and enables rollouts via rollout.py

### PR DESCRIPTION
**Pull request by divyahansg**
- `restore_checkpoint()` now is called after a learner is fully initialized
-  Rollouts are conducted via `rollout.py`

### CODE REVIEW

File: surreal/session/default_configs.py [hunk 1]
```diff
@@ -179,7 +179,7 @@
     'tensorplex': {
         'host': 'localhost',
         'port': 7008,
-        'tensorboard_port': 6006,
+        'tensorboard_port': 9009,
```
Comment (LinxiFan) [inline surreal/session/default_configs.py]: Is there a reason we use 9009? @amandlek

**Comment by LinxiFan** [comment]
@amandlek could you please help review this?

**Comment by jirenz** [comment]
The changes look fine. What command should I run to test the functionalities?

File: surreal/session/default_configs.py (hunk 1 above)
Comment (amandlek) [reply surreal/session/default_configs.py]: there was no reason - I just did it when testing to avoid conflict with some other stuff I had on 6006

**Comment by amandlek** [comment]
Also @LinxiFan the code looks identical to when I made the edits for enabling rollouts - so I don't think that I should be the one to review these changes given my lack of familiarity with the rest of the code. But @jirenz seems to approve of the changes. 

@jirenz I believe you can just use the rollout.py script on a checkpoint to test it out.

**Comment by jirenz** [comment]
@LinxiFan @divyahansg I remember I worked with Ajay's rollout commit before and it was working. @divyahansg what is the main difference here?

**Comment by LinxiFan** [comment]
@jirenz I think Divya verified that the checkpointing is working, and made some minor modifications. He didn't do much beyond that.

**Pull request merged**